File: AreaTriangulo.java
import java.util.Scanner;

 class AreaTriangulo{
  static Scanner teclado=new
  Scanner(System.in);
   
  public static void main(String[]arg){
   //definicion de variables u otros
   System.out.println("Ejercicio 01: Area Triangulo");
   int b=0, h=0, area=0;
   //datos de entrada
   System.out.println("Ingrese Base");
   b=teclado.nextInt();
   System.out.println("Ingrese Altura");
   h=teclado.nextInt();
   //proceso
   area=(b*h)/2;
   //datos de salida
   System.out.println("El Area es: "+area);
 }
}

File: AreaTriangulo.go
package main
import "fmt"
func main(){
  //declaracion de variables y otros
  fmt.Println("Ejercicio 01: Area Triangulo ")
  var b int
  var h int
  var area float64
  //datos de lectura
  fmt.Println("ingrese base:")
  fmt.Scanln(&b)
  fmt.Println("ingrese altura")
  fmt.Scanln(&h)
  //proceso
  area=float64((b*h)/2)
  //datos de salida
  fmt.Println("el area es: ",area)
}